demo/bootsfaces-applicant-portlet/src/main/webapp/WEB-INF/views/applicant.xhtml

```diff
@@ -44,12 +44,14 @@
 								</h:panelGroup>
 							</b:column>
 							<b:column id="c1b">
-								<h:panelGroup id="dateOfBirthField">
+								<h:panelGroup id="dateOfBirthField" binding="#{dateOfBirthField}">
 									<b:dateTimePicker id="dateOfBirth" label="#{i18n['birthday']}"
-										format="#{portletPreferencesValues['datePattern'][0].toUpperCase()}" required="true"
-										styleClass="date-picker" value="#{applicantModelBean.dateOfBirth}">
+										format="#{portletPreferencesValues['datePattern'][0].toUpperCase()}"
+										required="true" styleClass="date-picker" value="#{applicantModelBean.dateOfBirth}">
 										<f:convertDateTime pattern="#{portletPreferencesValues['datePattern'][0]}" />
-										<f:ajax render="dateOfBirthField" />
+										<!-- b:dateTimePicker normally uses the onclick attribute for ajax, so -->
+										<!-- manually specify ajax in the onchange attribute. -->
+										<f:attribute name="onchange" value="jsf.ajax.request(this,event,{execute:'#{dateOfBirthField.clientId}',render:'#{dateOfBirthField.clientId}'})" />
```

Comment:
@stiemannkj1 Would it be possible to specify `<f:ajax event="onchange" render="dateOfBirthField" />` instead?